src/widget/wsearchlineedit.cpp

```diff
@@ -19,7 +19,7 @@ const mixxx::Logger kLogger("WSearchLineEdit");
 
 const QColor kDefaultBackgroundColor = QColor(0, 0, 0);
 
-const QString kEmptySearch = QStringLiteral("");
+const QString kEmptySearch = QLatin1String("");
```

Comment:
Can be removed completely

Reply:
I was confused why this explicitly checks that the search string is not null, but there is not comment why this is necessary. I removed it and noticed not regression, but please give it a long, hard stare.